packages/cli/src/util/output/create-output.ts

```diff
@@ -43,7 +39,7 @@ export class Output {
   };
 
   print = (str: string) => {
-    if (this.textOnly) {
```

Comment:
**suggestion (blocking):** implement emoji removal in `emoji` function

Can we implement the emoji removal in [emoji.ts](https://github.com/vercel/vercel/blob/main/packages/cli/src/util/emoji.ts) directly instead of here?

Not all output goes through this Output class. We could miss cases that need removal by implementing it here.

Reply:
That's good idea.
Maybe if there is `--no-color` flag I should set some env var to reach no color mode in `prependEmoji` function.
Is it Okay?

Reply:
[d8651cc](https://github.com/vercel/vercel/pull/8826/commits/d8651ccad7c1dcc1ffabb9eaab952e93e830efa9)